### PR TITLE
Publishing my subscribed channels to the client

### PR DESCRIPTION
**Pull request by cjsauer**
What I did:
+ Created a 'myChannels' publication
+ Tested all of our publications thus far with Jasmine client tests
+ Moved our cucumber fixtures out into its own package, so that we can share them with Jasmine
+ Tiny tweaks to the User astro api

### CODE REVIEW

File: .meteor/versions [hunk 1]
```diff
@@ -12,10 +12,6 @@ blaze@2.1.3
 blaze-html-templates@1.0.1
 blaze-tools@1.0.4
 boilerplate-generator@1.0.4
-browser-policy@1.0.5
-browser-policy-common@1.0.4
-browser-policy-content@1.0.6
-browser-policy-framing@1.0.6
```
Comment (marti584) [inline .meteor/versions]: BrowserPolicy lets you set security-related policies that will be enforced by newer browsers. It's not necessarily important in our case that we don't have the package, but if problems were experienced when trying to put the instance inside of a frame, the documentation says to add 

```
BrowserPolicy.framing.allowAll();
```

**Comment by cjsauer** [comment]
:+1:

**Comment by marti584** [comment]
:+1:

**Comment by elGatoMantocko** [comment]
:+1:

**Comment by elGatoMantocko** [comment]
:boom:

**Pull request merged**